src/styles.css/style.css

```diff
@@ -0,0 +1,118 @@
+@import "./stars.css";
+@import "./reset.css";
+
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  --color-dark: #060b35;
+  --color-light: #616070;
+  --color-blue: #00acdc;
+}
+
+.card__image {
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__model {
+  display: block;
+  font-family: "Roboto", Arial, sans-serif;
+  color: var(--color-dark);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  padding: 0 16px;
+  margin-bottom: 4px;
+}
+
+.card__item-code {
+  font-family: "Roboto", Arial, sans-serif;
+  color: var(--color-light);
+  font-size: 10px;
+  font-weight: 400;
+
+  padding: 0 16px;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 16px;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  font-family: "Roboto", Arial, sans-serif;
```

Comment:
We don't need to repeat this font-family once more, we set it at html

Reply:
Fix it everywhere